arbor/fvm_lowered_cell_impl.hpp

```diff
@@ -136,6 +136,15 @@ class fvm_lowered_cell_impl: public fvm_lowered_cell {
     void set_gpu() {
         if (context_.gpu->has_gpu()) context_.gpu->set_gpu();
     }
+
+    // Translate cell probe descriptions into probe handles etc.
```

Comment:
Can the comment document the arguments.

Reply:
I think the only unclear one would be `mech_tbl`, right? If so, I can just change the name of the parameter. We otherwise only ever have the one `fvm_cv_discretization`, `fvm_mechanism_data`, and vector of `target_handle`.